components/create-collective/NewCreateCollective.js

```diff
@@ -0,0 +1,259 @@
+    const { category, form, error } = this.state;
+    const { token } = query;
+
+    const canApply = get(this.props.host, 'settings.apply') || true;
+
+    return (
+      <Page>
+        <div className="CreateCollective">
+          {canApply && !LoggedInUser && (
+            <Fragment>
+              <Flex flexDirection="column" alignItems="center" mb={5} p={2}>
+                <Flex flexDirection="column" p={4} mt={2}>
+                  <Box mb={3}>
+                    <H1 fontSize="H3" lineHeight="H3" fontWeight="bold" textAlign="center">
+                      Join Open Collective
+                    </H1>
+                  </Box>
+                  <Box textAlign="center">
+                    <P fontSize="Paragraph" color="black.600" mb={1}>
+                      Create an account (or sign in) to start a collective.
+                    </P>
+                  </Box>
+                </Flex>
+                <SignInOrJoinFree />
+              </Flex>
+            </Fragment>
+          )}
+          {canApply && LoggedInUser && !category && (
+            <CollectiveCategoryPicker query={query} onChange={(key, value) => this.handleChange(key, value)} />
+          )}
+          {canApply && LoggedInUser && category && category !== 'opensource' && (
+            <CreateCollectiveForm
+              host={this.props.host}
+              collective={this.state.collective}
+              onSubmit={this.createCollective}
+              onChange={(key, value) => this.handleChange(key, value)}
+              error={error}
+              query={query}
+            />
+          )}
+          {canApply && LoggedInUser && category === 'opensource' && !form && (
+            <ConnectGithub token={token} query={query} onChange={(key, value) => this.handleChange(key, value)} />
+          )}
+          {canApply && LoggedInUser && category === 'opensource' && form && (
+            <CreateCollectiveForm
+              host={this.props.host}
+              collective={this.state.collective}
+              onSubmit={this.createCollective}
+              onChange={(key, value) => this.handleChange(key, value)}
+              error={error}
+              query={query}
+            />
+          )}
+        </div>
+      </Page>
+    );
+  }
+}
+
+const createCollectiveQueryV2 = gqlV2`
+  mutation createCollective(
+    $collective: CreateCollectiveInput!
+    $host: AccountInput
+    $automateApprovalWithGithub: Boolean
+  ) {
+    createCollective(collective: $collective, host: $host, automateApprovalWithGithub: $automateApprovalWithGithub) {
+      name
+      slug
+      tags
+      description
+      githubHandle
+    }
+  }
+`;
+
+const addCreateCollectiveMutationV2 = graphql(createCollectiveQueryV2, {
+  options: {
+    context: { apiVersion: '2' },
+  },
+  props: ({ mutate }) => ({
+    createCollectiveV2: async ({ collective, host }) => {
+      const CreateCollectiveInputType = pick(collective, [
```

Comment:
Is `pick` really necessary here? What's in `collective` without picking?

Also, earlier we're doing `delete collective.tos;`, this is duplicate with picking. So, in case it's necessary, we should have a single strategy.

Reply:
Ok, will use `pick`